x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -267,27 +312,98 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
             />
           </EuiFormRow>
-          <EuiFormRow
-            isInvalid={createIndexPattern && destinationIndexPatternTitleExists}
-            error={
-              createIndexPattern &&
-              destinationIndexPatternTitleExists && [
-                i18n.translate('xpack.ml.dataframe.analytics.create.indexPatternTitleError', {
-                  defaultMessage: 'An index pattern with this title already exists.',
-                }),
-              ]
-            }
-          >
-            <EuiSwitch
-              disabled={isJobCreated}
-              name="mlDataFrameAnalyticsCreateIndexPattern"
-              label={i18n.translate('xpack.ml.dataframe.analytics.create.createIndexPatternLabel', {
-                defaultMessage: 'Create index pattern',
-              })}
-              checked={createIndexPattern === true}
-              onChange={() => setFormState({ createIndexPattern: !createIndexPattern })}
-            />
-          </EuiFormRow>
+          {jobType === JOB_TYPES.REGRESSION && (
+            <Fragment>
+              <EuiFormRow
+                label={i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.dependentVariableLabel',
+                  {
+                    defaultMessage: 'Dependent variable',
+                  }
+                )}
+                error={
+                  dependentVariableFetchFail === true && [
+                    <Fragment>
+                      {i18n.translate(
+                        'xpack.ml.dataframe.analytics.create.dependentVariableOptionsFetchError',
+                        {
+                          defaultMessage: 'There was a problem fetching fields. Please try again.',
+                        }
+                      )}
+                    </Fragment>,
+                  ]
+                }
+              >
+                <EuiComboBox
+                  aria-label={i18n.translate(
+                    'xpack.ml.dataframe.analytics.create.dependentVariableInputAriaLabel',
+                    {
+                      defaultMessage: 'Enter field to be used as dependent variable.',
+                    }
+                  )}
+                  placeholder={i18n.translate(
+                    'xpack.ml.dataframe.analytics.create.dependentVariablePlaceholder',
+                    {
+                      defaultMessage: 'dependent variable',
+                    }
+                  )}
+                  isDisabled={isJobCreated}
+                  isLoading={loadingDepFieldOptions}
+                  singleSelection={true}
+                  options={dependentVariableOptions}
+                  selectedOptions={dependentVariable ? [{ label: dependentVariable }] : []}
+                  onChange={selectedOptions =>
+                    setFormState({ dependentVariable: selectedOptions[0].label || '' })
+                  }
+                  isClearable={false}
+                  isInvalid={dependentVariable === ''}
```

Comment:
I wonder if we need this check left in, or if we should extend it to other fields on the form? It looks a bit odd that it is the only field that is flagged as being required, when in fact most of the others are too (job ID, source and dest index), but we handle those fields differently in only enabling the 'Create' button when they all have values.

Reply:
Good one, @peteharverson 🤔 Agree that it should follow the same pattern of validation. I'll add it to the follow-up if that sounds alright 👌